internal/api/ratelimits_test.go

```diff
@@ -19,14 +19,25 @@ func Example_newRateLimiter() {
 		cfg := conf.Rate{Events: 100, OverTime: time.Hour * 24}
 		rl := newRateLimiter(cfg)
 		cur := now
-		for i := 0; i < 61; i++ {
-			fmt.Printf("%-5v @ %v\n", rl.AllowN(cur, 1), cur)
-			cur = cur.Add(time.Minute)
+		for limited, i := 0, 0; i < 160; i++ {
+			allowed := rl.AllowN(cur, 1)
+			if allowed {
+				limited++
+			}
+
+			switch {
+			case i == 100:
+				fmt.Printf("true  @ %v for last %v events...\n", cur, limited)
+			case i > 100:
+				fmt.Printf("%-5v @ %v\n", allowed, cur)
```

Comment:
Maybe remove these :D

Reply:
Or maybe better yet, dump them in a big string with Sprintf and then make sure the expected and actual strings match.

Reply:
This is inside a Example, so go will verify the lines under `// Output:` as you describe here.